src/k8s/common.ts

```diff
@@ -20,4 +28,16 @@ export async function getQuickPicks(cmd: string, errorMessage: string, converter
 export async function selectResourceByName(config: Promise<QuickPickItem[]> | QuickPickItem[], placeHolderText: string): Promise<string> {
     const resource: any = await window.showQuickPick(config, {placeHolder: placeHolderText});
     return resource ? resource.label : null;
+}
+
+export async function getNode(command: string, text1: string, text2: string): Promise<k8s.ClusterExplorerV1.Node[]> {
```

Comment:
should be named `getNodes` because it returns array. 
Please use self explanatory names. `text1` and `text2` gives no clue what should be passed to this function.
Should be 'kind` and `abbreviation` instead.